packages/vue/src/components/organisms/SfMegaMenu/_internal/SfMegaMenuColumn.html

```diff
@@ -0,0 +1,8 @@
+<div v-if="isActive || !isMobile" class="sf-mega-menu-column">
```

Comment:
```suggestion
<div class="sf-mega-menu-column">
```
Propably we don't need `v-if="isActive || !isMobile"`

Reply:
we do, otherwise other columns will be shown